Tests/DefaultXpringClientTest.swift

```diff
@@ -68,6 +68,124 @@ final class DefaultXpringClientTest: XCTestCase {
     }
   }
 
+  // MARK: - Send
+
+  func testSendWithSuccess() {
+    // GIVEN a Xpring client which will successfully return a balance from a mocked network call.
+    let xpringClient = DefaultXpringClient(networkClient: FakeNetworkClient.successfulFakeNetworkClient)
+
+    // WHEN XRP is sent.
+    guard
+      let transactionHash = try? xpringClient.send(
+        .sendAmount,
+        to: .destinationAddress,
+        from: .wallet)
+      else {
+        XCTFail("Exception should not be thrown when trying to send XRP")
+        return
+    }
+
+    // THEN the engine result code is as expected.
+    XCTAssertEqual(transactionHash, TransactionHash.testTransactionHash)
+  }
+
+  func testSendWithClassicAddress() {
+    // GIVEN a classic address.
+    guard let classicAddressComponents = Utils.decode(xAddress: .destinationAddress) else {
+      XCTFail("Failed to decode X - Address.")
+      return
+    }
+    let xpringClient = DefaultXpringClient(networkClient: FakeNetworkClient.successfulFakeNetworkClient)
+
+    // WHEN XRP is sent to a classic address THEN an error is thrown.
+    XCTAssertThrowsError(try xpringClient.send(
```

Comment:
Why are we cutting functionality to send to classic addresses? Just curious

Reply:
This was a product level decision that we wanted people to use the newest features the ecosystem had to offer.